
syntax = "proto2";
message SidechainRegistrationParams {
    required bytes chainID = 1;
    required string name = 2;
    repeated SidechainRegistrationParams_SidechainValidators sidechainValidators = 3;
    required uint64 sidechainCertificateThreshold = 4;
    
}

message SidechainRegistrationParams_SidechainValidators {
    required bytes blsKey = 1;
    required uint64 bftWeight = 2;
    
}
